nifi-nar-bundles/nifi-social-media-bundle/nifi-twitter-processors/src/main/java/org/apache/nifi/processors/twitter/ConsumeTwitter.java

```diff
@@ -255,6 +258,8 @@ public class ConsumeTwitter extends AbstractProcessor {
 
     private volatile BlockingQueue<String> messageQueue;
 
+    private final AtomicBoolean isInitialized = new AtomicBoolean(false);
```

Comment:
Recommend adjusting the name for clarity:
```suggestion
    private final AtomicBoolean streamStarted = new AtomicBoolean(false);
```

Reply:
Making the changes